gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapper.java

```diff
@@ -0,0 +1,242 @@
+package uk.nhs.adaptors.pss.translator.mapper;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.hl7.fhir.dstu3.model.Annotation;
+import org.hl7.fhir.dstu3.model.CodeType;
+import org.hl7.fhir.dstu3.model.CodeableConcept;
+import org.hl7.fhir.dstu3.model.Coding;
+import org.hl7.fhir.dstu3.model.Condition;
+import org.hl7.fhir.dstu3.model.DateTimeType;
+import org.hl7.fhir.dstu3.model.Extension;
+import org.hl7.fhir.dstu3.model.IdType;
+import org.hl7.fhir.dstu3.model.Identifier;
+import org.hl7.fhir.dstu3.model.Meta;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.fhir.dstu3.model.Resource;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.hl7.fhir.dstu3.model.StringType;
+import org.hl7.fhir.dstu3.model.UriType;
+import org.hl7.v3.CD;
+import org.hl7.v3.IVLTS;
+import org.hl7.v3.RCMRMT030101UK04Annotation;
+import org.hl7.v3.RCMRMT030101UK04Component4;
+import org.hl7.v3.RCMRMT030101UK04EhrComposition;
+import org.hl7.v3.RCMRMT030101UK04LinkSet;
+import org.hl7.v3.RCMRMT030101UK04ObservationStatement;
+import org.hl7.v3.RCMRMT030101UK04PertinentInformation02;
+import org.hl7.v3.TS;
+
+import lombok.AllArgsConstructor;
+
+@AllArgsConstructor
+public class ConditionMapper {
+    private static final String META_PROFILE = "https://fhir.nhs.uk/STU3/StructureDefinition/CareConnect-GPC-ProblemHeader-Condition-1";
+    private static final String RELATED_CLINICAL_CONTENT_URL = "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-RelatedClinicalContent-1";
+    private static final String IDENTIFIER_SYSTEM = "https://PSSAdaptor/{practiseCode}";
+    private static final String PRACTISE_CODE = "{practiseCode}";
+    private static final String ACTUAL_PROBLEM_URL = "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ActualProblem-1";
+    private static final String PROBLEM_SIGNIFICANCE_URL = "https://fhir.hl7.org.uk/STU3/StructureDefinition/Extension-CareConnect-ProblemSignificance-1";
+    private static final String MAJOR_CODE = "386134007";
+    private static final String CLINICAL_STATUS_ACTIVE_CODE = "394774009";
+    private static final String CLINICAL_STATUS_INACTIVE_CODE = "394775005";
+    private static final String DEFAULT_CLINICAL_STATUS = "Defaulted status to active : Unknown status at source";
+
+    private CodeableConceptMapper codeableConceptMapper;
```

Comment:
Sorry Elliot, I should have done it earlier, but could you make CodeableConceptMapper a spring `@Service` and then inject it here? And DateTimeMapper as well. And then inside test you could annotate ConditionMapper with @InjectMocks (maybe it'll simplify things).